com.chartboost.helium/Android~/helium-android-bridge/src/main/kotlin/com/chartboost/heliumsdk/unity/HeliumEventProcessor.kt

```diff
@@ -0,0 +1,95 @@
+package com.chartboost.heliumsdk.unity
+
+import android.util.Log
+import com.chartboost.heliumsdk.ad.HeliumAdError
+import org.json.JSONException
+import org.json.JSONObject
+
+object HeliumEventProcessor {
+    private const val TAG = "HeliumEventProcessor"
+    private const val EMPTY_STRING = ""
```

Comment:
pretty sure empty string should never be a thing... 
using `""` is perfectly fine.

Reply:
Yes, EMPTY_STRING was something I was using from C# since there isn't a string.Empty version in JAVA and it was getting used as a default value all over the place, after refactoring is not less common so I have removed it.